src/chrome/content/rules/Deepin.xml

```diff
@@ -0,0 +1,49 @@
+<ruleset name="Deepin (partial)">
+    <!-- Deepin.com -->
+    <target host="www.deepin.com" />
+    <!-- Redirects to *.linuxdeepin.com: -->
+    <target host="packages.deepin.com" />
+    <target host="cdimage.deepin.com" />
+    
+    <!-- Deepin.org -->
+    <target host="www.deepin.org" />
+    <target host="bbs.deepin.org" />
+    <target host="blog.deepin.org" />
+    <target host="api.deepin.org" />
+    <target host="account.deepin.org" />
+    <!-- Redirects to *.linuxdeepin.com: -->
+    <target host="packages.deepin.org" />
+    
+    <!-- Linuxdeepin.com -->
+    <target host="img1.linuxdeepin.com" />
+    <target host="img2.linuxdeepin.com" />
+    <target host="packages.linuxdeepin.com" />
+    <target host="cdimage.linuxdeepin.com" />
+    
+    <!-- Redirects to *.linuxdeepin.com -->
+    <rule from="^http://(packages|cdimage)\.deepin\.com/" to="https://$1.linuxdeepin.com/" />
+    <rule from="^http://(packages)\.deepin\.org/" to="https://$1.linuxdeepin.com/" />
+    <!-- Redirects directly to HTTPS -->
+    <rule from="^http:" to="https:" />
+    
+    <!--
+    Mismatch:
+    wiki.deepin.org
+    blog.deepin.org
```

Comment:
Seems to working now (https://github.com/EFForg/https-everywhere/pull/4230/files#diff-b460b3e9ba52aa834775ba7d1b087c00R11)